components/icon/icon.twig

```diff
@@ -35,12 +35,16 @@
 {% set theme = theme|default('localgov_base') %}
 {% set icon_wrapper_element = icon_wrapper_element|default('div') %}
 
+{%- set icon_fullpath -%}
+  {% if icon_path|first != '@' %}@{{ theme }}/{% endif %}{{ icon_path|default('includes/icons') }}/{{ icon_name }}.svg
+{%- endset -%}
+
 {% set attributes = create_attribute()
   .setAttribute('aria-hidden', decoration|default('true'))
   .addClass('lgd-icon')
   .addClass(icon_classes)
 %}
 
 <{{ icon_wrapper_element }}{{ attributes }}>
-  {% include icon_path ~ '/' ~ icon_name ~ '.svg' %}
+  {% include icon_fullpath %}
```

Comment:
Let's change this line to 
`{% if icon_path|first != '@' %}@{{ theme }}/{% endif %}{{ icon_path|default('includes/icons') }}/{{ icon_name }}.svg`

_after_ we remove the `set` from above, like so:

```
<{{ icon_wrapper_element }}{{ attributes }}>
  {% if icon_path|first != '@' %}@{{ theme }}/{% endif %}{{ icon_path|default('includes/icons') }}/{{ icon_name }}.svg
</{{ icon_wrapper_element }}>
```

Reply:
That will be nicer. I'll do that this afternoon.

Reply:
"This afternoon" came today for me :roll_eyes: